File: src/main/java/seremis/geninfusion/soul/gene/newAI/play/GeneAIPlay.java
package seremis.geninfusion.soul.gene.newAI.play;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.SoulHelper;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAIPlay extends MasterGene {

    public GeneAIPlay() {
        addControlledGene(Genes.GENE_AI_PLAY_INDEX);
        addControlledGene(Genes.GENE_AI_PLAY_MOVE_SPEED);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/scala/seremis/geninfusion/soul/gene/newAI/GeneAIPlay.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneAIPlay extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GeneAIPlayIndex)
    addControlledGene(Genes.GeneAIPlayMoveSpeed)
}
